pkg/operator/controller/controller_dns_daemonset_test.go

```diff
@@ -497,6 +498,36 @@ func TestDaemonsetConfigChanged(t *testing.T) {
 				}
 				daemonset.Spec.Template.Annotations["cluster-autoscaler.kubernetes.io/enable-ds-eviction"] = "true"
 			},
+			expect: false, // it is already there, and true
```

Comment:
This is the same as the "if nothing changes" test case then, right?

Reply:
Well, yes.  But it was `expect: true` before, and I guess I wanted to highlight the difference with the changes in this PR.  I can remove it.  Or did you want me to add a different test?

Reply:
@Miciah I fixed the tests. PTAL